node/worker_threads.ts

```diff
@@ -126,6 +126,7 @@ interface NodeEventTarget extends
   removeListener: NodeEventTarget["off"];
 }
 
+// TODO(duelsik): Post an issue about making this place less imperative and just export different class depending on `isMainThread`
```

Comment:
I'm not sure I understand this comment. Is this a TODO to open a GitHub issue? If so, I think it can be removed from here.